src/libraries/System.Runtime/tests/System/TimeZoneInfoTests.cs

```diff
@@ -2314,7 +2314,7 @@ public static IEnumerable<object[]> SystemTimeZonesTestData()
                 for (int i = -14; i <= 12; i++)
                 {
                     TimeZoneInfo tz = null;
- 
+
                     try
                     {
                         string id = $"Etc/GMT{i:+0;-0}";
```

Comment:
This formatting is going to be subject to the current culture, which could impact the negative sign used.  Is that going to be an issue for someone running the tests in, say, Sweden?

Reply:
It is possible to fail. As this is not related to this PR, we can address that in another PR.

---
In reply to: [611643769](https://github.com/dotnet/runtime/pull/51055#discussion_r611643769) [](ancestors = 611643769)